pkg/acs/calacs/acsrej/rej_command.c

```diff
@@ -89,8 +104,22 @@ int newpar[];       o: array of parameters set by the user
                 par->shadcorr = 1;
                 ctoken++;
 
+            // This option/variable was previously named "newbias", and it has been renamed
+            // to "readnoise_only" which is more informative.  The variable controls
+            // the type of noise used in the computation of the error (ERR) array.  In particular,
+            // this variable is intended to control the noise used when processing BIAS data.  When
+            // using CALACS, readnoise_only will only be set to "1" (true) for BIAS data.  When using
+            // ACSREJ as a standalone utility, the readnoise_only can be set to "1" for any data.
             } else if (strcmp("newbias", argv[ctoken]+1) == 0) {
-                par->newbias = 1;
+                printf ("\n*************************************************************************************\n");
+                printf ("     As of HSTCAL Build 2018.1, the 'newbias' option for ACSREJ has been renamed \n");
```

Comment:
I'm not sure of our official stance on this but we may need to refer to HSTCAL by something other than our internal build identifier (2018.1), i.e. the actual version number. This definitely needs to be done, I'm just not sure if we need both or just the latter. It may even need to also reference the version of CALACS.